std/algorithm/comparison.d

```diff
@@ -1565,3 +1568,228 @@ unittest
                 2, "two",
                 ));
 }
+
+/**
+Returns true if all of the elements in $(D r1) are present in $(D r2). Otherwise,
+returns false.
+
+If your parameters are forward ranges, then this function can allocate if the
+$(D AllocateGC.yes) flag is passed. This has the benefit of being much
+faster than the $(D AllocateGC.no) option.
```

Comment:
Maybe we should change "much faster" to "have better complexity". I can't see that allocating with the GC will be much faster for small ranges.